agent/host_agent_test.go

```diff
@@ -129,7 +172,7 @@ var _ = Describe("Agent", func() {
 					return nil
 				}
 				return createdByoHost
-			}).ShouldNot(BeNil())
+			}, 30).ShouldNot(BeNil())
```

Comment:
The registration should not take this long IMO. Have you tried without the extended timeout?